Clarify return values for Text Input components in an interaction (#6344)
hammerandchisel/discord-api-docs: docs/interactions/Message_Components.md

> Text inputs are an interactive component that render on modals. They can be used to collect short-form or long-form text.
> 

Reviewer: ```suggestion
Text inputs are an interactive component that render in modals. They can be used to collect short-form or long-form text.

```
sorry i know your PR doesn't touch this, but while I'm here lol

> When defining a text input component, you can set attributes to customize your component that are not returned in the [text input interaction payload](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/text-input-object-text-input-interaction). These can be used to customize whether the text input is required and how it is visually displayed to users.

Reviewer: ```suggestion
When defining a text input component, you can set attributes to customize the behavior and appearance of it. However, not all attributes will be returned in the [text input interaction payload](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/text-input-object-text-input-interaction).
```

Feel free to modify—just trying to clean up wording a little bit.
